docs/src/components/ServiceTest.tsx

```diff
@@ -160,21 +154,20 @@ export default function ServiceTest(props: {
                                 />
                             </StepContent>
                         </Step>
-                        {serviceTest?.tests.map((test, index) => (
+                        {testRunner?.tests.map((test, index) => (
                             <Step key={index}>
```

Comment:
this should be moved into a separate component;'s generate easier when you refactor those.

Reply:
I know - one aspect is that this generates a sequence of Step... if I componentize, I need a parent. Just use a <div>?

Reply:
Keep the "<Step>" part in the function, and move the invisde of it in a component